react-app/src/components/Banner/MMDLSpaBanner.tsx

```diff
@@ -1,12 +1,22 @@
-import { useState } from "react";
 import { Cross2Icon } from "@radix-ui/react-icons";
 import { InfoCircledIcon } from "@radix-ui/react-icons";
+import { useNotifs } from "@/hooks";
+import { useGetSystemNotifs } from "@/api";
+
 const MMDLAlertBanner = () => {
-  const [isVisible, setIsVisible] = useState(true);
+  const { dismissed, clearNotif } = useNotifs();
+  const result = useGetSystemNotifs();
```

Comment:
Should we consolidate these two hooks? Will they ever be used independently? 

If yes, we can definitely move all this logic into the hook and let the component just concern itself with presentation